lib/themes/dosomething/paraneue_dosomething/scss/patterns/_button.scss

```diff
@@ -1,14 +1,73 @@
+%animated {
```

Comment:
it's the [placeholder selector!](http://thesassway.com/intermediate/understanding-placeholder-selectors).